todo_app/flask_config.py

```diff
@@ -8,4 +8,10 @@ class Config:
         raise ValueError("No SECRET_KEY set for Flask application. Did you forget to run setup.sh?")
     LOGIN_DISABLED = os.environ.get('LOGIN_DISABLED')
     if not LOGIN_DISABLED:
-        LOGIN_DISABLED = False
\ No newline at end of file
+        LOGIN_DISABLED = False
+    LOG_LEVEL = os.environ.get('LOG_LEVEL')
+    if not LOG_LEVEL:
+        LOG_LEVEL = "ERROR"
+    LOGGLY_TOKEN = os.environ.get('LOGGLY_TOKEN')
+    if not LOGGLY_TOKEN:
+        raise ValueError("No Loggly token set")
```

Comment:
Your app.py acts like LOGGLY_TOKEN is optional, but then here you throw an error if it's not set.